docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -539,6 +537,105 @@ jobs:
 
 </Collapsible>
 
+## TestFlight
+
+Distribute iOS builds to TestFlight internal and external testing groups. This is an alternative to the iOS submit job for when you need more advanced TestFlight features. If you need to control test groups, changelog, or Beta App Review submission, use the testflight job instead of submit.
```

Comment:
```suggestion
Distribute iOS builds to TestFlight internal and external testing groups. This is an alternative to the iOS submit job for when you need more advanced TestFlight features. If you need to control test groups, changelog, or Beta App Review submission, use the `testflight` job instead of submit.
```

Since "testflight" is a keyword and has a significance in code syntax.